### PR TITLE
[2.0] change package priority to 11

### PR DESCRIPTION
**Pull request by TaykYoku**

### CODE REVIEW

File: src/COMDIRAC/__init__.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 
 
 def extension_metadata():
-    return {"priority": 10}
+    return {"priority": 11}
```
Comment (chrisburr) [inline src/COMDIRAC/__init__.py:15]: Does this need to be an extension at all? It doesn't override anything does it?

Comment (chrisburr) [reply src/COMDIRAC/__init__.py:15]: To be clear, I mean it doesn't need to be a "DIRAC extension" and can just be a standard python package that depends on `DIRAC`.

Comment (TaykYoku) [reply src/COMDIRAC/__init__.py:15]: this package also has an `Interfaces/scripts` directory for ex.

So this function is not needed? can i delete it?

Comment (chrisburr) [reply src/COMDIRAC/__init__.py:15]: Ah it's using the DIRAC argument parser. In that case it's probably easier to leave it as is.

**Pull request merged**